swan-lake/integration-tools/health-tool.md

```diff
@@ -16,75 +16,128 @@ They are primarily focused on the [Fast Healthcare Interoperability Resources (F
 
 Download a directory containing all FHIR specification definition files. You can download a preferred standard implementation guide from the [FHIR Implementation Guide registry](https://fhir.org/guides/registry/).
 
->**Note:** It is recommended to use a Standard for Trial Use (STU) or higher release level of the implementation guides. Make sure that the downloaded specification archive has the StructureDefinition, ValueSet, and CodeSystem files for the Implementation Guide (IG) resources when extracted.
+>**Note:** It is recommended to use a Standard for Trial Use (STU) or higher release level of the implementation guides. Make sure that the downloaded specification archive has the `StructureDefinition`, `ValueSet`, and `CodeSystem` files for the Implementation Guide (IG) resources when extracted.
+
+## Install the tool
+
+Execute the command below to pull the Health tool from [Ballerina Central](https://central.ballerina.io/search?q=ballerinax%2Fhealth).
+
+```
+$ bal tool pull health
+health:2.0.0 pulled successfully.
+health:2.0.0 successfully set as the active version.
+```
 
 ## Usage
 
-This tool uses the command below to generate a Ballerina package for a given FHIR implementation guide. 
+The Ballerina Health tool support provides the main capabilities below.
+
+- [**Package generation:**](#package-generation) generate a Ballerina package from a given FHIR implementation guide.
+- [**Template generation:**](#template-generation) generate a Ballerina API template from a given FHIR implementation guide.
+
+The general usage of the tool is as follows.
 
 ```
-$ bal health fhir -m <Mode: package> [OPTIONS] <fhir-specification-directory-path>
+$ bal health fhir -m <Mode: package|template> [OPTIONS] <fhir-specification-directory-path>
```

Comment:
Do we need `Mode: part`

i.e. Can't it be `bal health fhir -m <package|template> [OPTIONS] <fhir-specification-directory-path>`

@nirmal070125 ?

Reply:
package , template are exact values so should we represent them bit differently to the <fhir spec directory..>? I agree that Mode: looks odd.

Reply:
I think it should be ok to have `bal health fhir -m <package|template> [OPTIONS] <fhir-specification-directory-path>` as we are explaining all the details in the tables.